### DISCUSSION TITLE
How to use different QFieldCloud server URL

### DISCUSSION THREAD

**Discussion by idcc-cursor** [opengisch/QField#2946] [Q&A]
Hi all
Is there a way to set a own qfield.cloud server adress/IP by Qfield App cloud login ?

Or does I have to compile the source by myself ... which does't work

**Comment by suricactus** [comment]
Double tap the Nyuki icon both in QField or QFieldSync and a new URL input will appear.

**Comment by idcc-cursor** [comment]
Thank you !! I like this community :-) and this piece of software
